app/routes/static_pages.js

```diff
@@ -43,10 +43,14 @@ export default Ember.Route.extend({
   actions: {
     back: function() {
       let backLink = this.get("backLink");
-      if (backLink) {
-        this.transitionTo(backLink);
+      if (!backLink) {
+        window.history.back();
+        return;
+      }
+      if (backLink.includes("request_purpose")) {
```

Comment:
const transitionPath = backLink.includes("request_purpose") ? 'home' : backLink;
this.transitionTo(transitionPath);

Reply:
I've reverted this change. Wont be required now.